opm/simulators/utils/readDeck.cpp

```diff
@@ -344,8 +352,6 @@ void readDeck(int rank, std::string& deckFilename, std::unique_ptr<Opm::Deck>& d
         errorGuard->clear();
     }
 
-
-    auto comm = Dune::MPIHelper::getCollectiveCommunication();
     parseSuccess = comm.min(parseSuccess);
```

Comment:
My bad, I didn't see the implication here.  Moving the definition of `comm` into the `HAVE_MPI` section means that
```C++
parseSuccess = comm.min(parseSuccess);
```
becomes invalid in sequential builds.  We should probably protect the `comm.min()` call with `#if HAVE_MPI` too.